src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const partnerSex = person.sex === 'f'
+    ? 'husband'
+    : 'wife';
+  const personIsMarried = person.isMarried
```

Comment:
For now you've added variable name the same as the properties name
So it's not clear what is the difference between `isMarried` property and `personIsMarried` variable